gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-common/src/main/java/io/gravitee/am/gateway/handler/common/email/impl/EmailServiceImpl.java

```diff
@@ -60,7 +69,10 @@
  * @author GraviteeSource Team
  */
 public class EmailServiceImpl implements EmailService, InitializingBean {
+    private static final Logger logger = LoggerFactory.getLogger(EmailServiceImpl.class);
 
+    public static final String CLIENT_ID = "client_id";
+    public static final String REDIRECT_URI = "redirect_uri";
```

Comment:
**todo:** We should use the constants defined into the oauth2.Parameters class https://github.com/gravitee-io/gravitee-access-management/blob/master/gravitee-am-common/src/main/java/io/gravitee/am/common/oauth2/Parameters.java

Reply:
done